bitwarden/values.yml

```diff
@@ -1,19 +1,17 @@
-replicaCount: 1
+deploy_version: 0.1.6
```

Comment:
Why do we have both `.yml` and `.yaml` values

Reply:
`.yml` was an accidental leftover. Deleted.